tests/README.md

```diff
@@ -3,18 +3,6 @@
 ## Introduction
 
 E2E UI tests
-
-### Test tools and frameworks
-
-- Selenium 4.x
-- Selenide 6.x
-- Cucumber 7.x
```

Comment:
its ok to drop versions but I think its worth mentioning what testing frameworks are used/required for new contributors.